keylime/cloud_verifier_tornado.py

```diff
@@ -289,6 +301,9 @@ def post(self):
                     agent_data['enc_alg'] = ""
                     agent_data['sign_alg'] = ""
                     agent_data['agent_id'] = agent_id
+                    agent_data['verifier_id'] = config.get('cloud_verifier', 'cloudverifier_id')
+                    agent_data['verifier_ip'] = config.get('cloud_verifier', 'cloudverifier_ip')
+                    agent_data['verifier_port'] = config.get('cloud_verifier', 'cloudverifier_port')
```

Comment:
If someone has an RPM of keylime installed the config file that the user may have modified may not get updated and then won't have these variables. So, I think we need a variant of `config.get()` that returns a given default value if the parameter is not in the config file...

Reply:
I agree, this has come up a couple of times now.

Reply:
I can send a patch with following the following pattern:

```
def get(section, option):
+def get(section, option, fallback=None):
+    if fallback is not None:
+        return get_config().get(section, option, fallback=fallback)
```

Reply:
PR https://github.com/keylime/keylime/pull/674

Reply:
@apoorvemohan can you rebase your work on this (now merged) change so that it's backwards compatible for any existing keylime.conf

Reply:
Rebased the work with the merged changes.